### DISCUSSION TITLE
openjverein für eine Solidarische Landwirtschaft

### DISCUSSION THREAD

**Discussion by budda85** [openjverein/jverein#327] [Q&A]
Hallo zusammen,
ich gucke mir gerade jamaica und openjverein an, um es eventuell in einer Solidarische Landwirtschaft zu nutzen.
Als gemeinnütziger Verein verwalten wir Geld für die erzeugenden Betriebe treuhändisch. Das heißt, das wir das Geld von unseren Mitglieder einsammeln und dann 1 zu 1 weiterleiten an die Betriebe.
Wir haben aktuell ca. 5 verschiedene Anteile, für welche wir auch Rechnungen im Namen der Betriebe erstellen. Auf den Rechnungen muss teilweise die Umsatzsteuer ausgewiesen werden. Mal mit 7% und mal mit 19%.
Wenn ich das aktuell richtig verstehe könnte ich das über die Zusatzbeträge abbilden. Ist das Richtig?

Dann werden bei uns in der Stadt mehrere Abholdepots betrieben. Über eigene Felddefinitionen könnte ich hier ja ein Feld erstellen, wo der Abholort pro Mitglied eingetragen wird.
Jetzt werden an den Depots Listen ausgelegt auf welchen abgehakt werden kann, ob man da war. Könnte ich mir irgendwie die Daten aus Datenbank raus holen? Also das ich filtern kann, welche Mitglieder z. B. Gemüse haben und an Abholtort 1 ihr Gemüse abholen. Oder eventuell sogar direkt ein Formular erstellen kann?

**Comment by JohannMaierhofer** [comment]
In den Einstellungen gibt es unter Buchführung die Option "Umsatzsteuer Option". Ist die an, kann man bei Buchungsarten einen Steuersatz eingeben. Allerdings habe ich das noch nie benutzt und weiß nicht was man da alles dann machen kann.

Wegen der Abholorte und Gemüse könnte man jeweils ein Zusatzfeld definieren. Dann kann man in der Mitgliederliste danach filtern.
Im Menü für freie Formulare kann man ebenfalls danach filtern und die Mitglieder dann in selbst definierte Formulare drucken.

**Comment by budda85** [comment]
Danke für deine Antwort. 
Das Gemüse kann drei verschiedene Preise haben (Normal, Ermäßigt und Soli). Von daher ist das Zusatzfeld, glaube ich, nicht das richtige dafür oder?
Zumal man auch mehrere Anteile buchen kann. Daneben kann man noch Eier, Kartoffeln und Fleisch buchen. Das ist alles frei wählbar und für jedes Mitglied individuell. Das einzige was verpflichtend ist, ist unser Mitgliedsbeitrag.

**Comment by JohannMaierhofer** [comment]
Ich verstehe ja noch nicht ganz den Anwendungsfall. Aber man könnte Gemüse normal, Gemüse ermäßigt, Genüse Soli, Eier, Kartoffeln etc. als Zusatzfeld vom Typ Integer anlegen. Dann kann man da die Anzahl bei jedem Mitglied separat eintragen. Im Zusatzfelder Filter kann man dann den Bereich (von, bis) angeben nach dem man filtern will, also z.B. von >0 für alle die da etwas stehen haben.
Die Felder kann man sich auch in der Mitgliederliste als Spalten anzeigen lassen (in Einstellungen Mitglieder Spalten auswählen) und in den Formularen ausgeben lassen (einfach als Formularfeld einfügen).

**Comment by lenilsas** [comment]
Wir haben auch eine SoLawi und wir machen es folgendermaßen: Für die Gemüseanteile habe ich Beitragsgruppen angelegt Normal und Ermäßigt in meinem Fall. Die zusätzlichen Produkte habe ich über Sekundäre Beitragsgruppen realisiert, hier kann bei jedem Mitglied eine oder mehrere davon gewählt werden. Wenn es Mitglieder ohne Gemüse gibt habe ich noch eine Beitragsgrupe "beitragsfrei" hier werden nur die Sekundären Beitragsgruppen bezahlt.
Für die Depots würde ich eine Eigenschaftengruppe "Depot" vorschlagen mit allen Depots als Eigenschaft, so kannst du bei jedem Mitglied wählen zu welchem Depot er gehört.
Für die Abhakliste kannst du unter Auswertungen->Mitglieder die Daten filtern und als CSV Datei exportieren. Ausgabe von Adresslisten sind auch direkt als PDF möglich.
Rechnungen habe ich bisher noch nicht in JVerein erstellt, ob da die Steuer mit ausgewiesen werden kann, kann ich gerade nicht sagen.

**Comment by budda85** [comment]
Die sekundären Beitragsgruppen hatte ich noch gar nicht gesehen. Gefällt mir, dass ich danach auch sortieren kann. Hier müsste ich mir nur dann vermutlich auch Beitragsgruppen anlegen, wenn jemand z. B. zwei oder drei Anteile eines bestimmten Anteils nimmt.
Kann ich es irgendwie feiner steuern, von welcher Beitragsgruppe ich eine Rechnung erstellen möchte?
Das Gemüse und Co wird bei uns monatlich eingezogen, der Vereinsbeitrag aber jährlich. Wenn ich den Vereinsbeitrag aber als Beitragsgruppe erstellen, wird mir der mit abgebucht, wenn ich über "Abrechnung" die Rechnungen erstelle.

Der Charme der Zusatzbeiträge ist, das man diese importieren kann. Einmal im Jahr werden bei uns die Anteile neu vergeben und da kommen so ca 250 verschiedene zusammen und da könnte man sich die Handarbeit sparen, da die Daten als Calc/Csv datei vorliegen. Da fehlt mir dann nur die Filtermöglichkeit...
Einen Tod muss ich vermutlich sterben :)

Die Eigenschaften für die Depots zu nehmen finde ich auch eine gute Idee. Besser als der Freitext in einem Zusatzfeld.
Steuern habe ich auch hin bekommen, das die angezeigt und berechnet werden. In eine Rechnung bekomme ich die auch.

**Comment by JohannMaierhofer** [comment]
Wegen der monatlichen Abbuchungen wird das wohl nur mit den Zusatzbeiträgen gehen.
Für die Filterung und Sortierung könnte man wie ich oben beschrieben habe zusätzlich Zusatzfelder anlegen. Allerdings muss man dafür die Informationen doppelt pflegen was nicht so schön ist.
Auch kann man die nicht importieren. Das wäre wohl noch ein sinnvolles Feature für Jverein.
